docs/UserGuide.md

```diff
@@ -360,6 +368,9 @@ Furthermore, certain edits can cause the MyBookshelf to behave in unexpected way
 
 ### Archiving data files
 
+### Introducing unique id for books
```

Comment:
I think this should be above ### Archiving data files, as this _Details coming soon_ should be the desription of ### Archiving data files